src/lib/types.ts

```diff
@@ -1,4 +1,4 @@
-import type { Options } from "mdast-util-toc"
+import { type Options, toc } from "mdast-util-toc"
```

Comment:
Revert this? it doesn't seem that we are using `toc`
```suggestion
import type { Options } from "mdast-util-toc"
```